resources/js/app.js

```diff
@@ -7,11 +7,13 @@ import calendar from './partials/calendar'
 import IMask from 'imask'
 import io from 'socket.io-client'
 import chat from './chat'
+import scrollToId from './components/scrollToId'
 
 Alpine.data('navbar', navbar)
 Alpine.data('alert', alert)
 Alpine.data('calendar', calendar)
 Alpine.data('chat', chat)
+Alpine.data('scrollToId', scrollToId)
```

Comment:
Use [alpine-magic-helpers](https://github.com/alpine-collective/alpine-magic-helpers#scroll) to implement the scroll